src/accounts/context/userAccount.tsx

```diff
@@ -78,32 +79,24 @@ export const UserAccountProvider: FC<Props> = React.memo(({children}) => {
 
   const handleGetAccounts = useCallback(async () => {
     try {
-      const resp = await getAccounts({})
-      if (resp.status !== 200) {
-        // set user account status to error;...TODO
-        throw new Error(resp.data.message)
+      const accounts = await getUserAccounts()
+      setUserAccounts(accounts)
+      const defaultAcctArray = accounts.find(acct => acct.isDefault)
+      if (defaultAcctArray) {
+        const defaultId = defaultAcctArray[0].id
+        setDefaultAccountId(defaultId)
       }
-      const {data} = resp
-      if (Array.isArray(data)) {
-        setUserAccounts(data)
-
-        const defaultAcctArray = data.filter(line => line.isDefault)
-        if (defaultAcctArray && defaultAcctArray.length === 1) {
-          const defaultId = defaultAcctArray[0].id
-          setDefaultAccountId(defaultId)
-        }
-
-        // isActive: true is for the currently logged in/active account
-        const activeAcctArray = data.filter(line => line.isActive)
-        if (activeAcctArray && activeAcctArray.length === 1) {
-          const activeId = activeAcctArray[0].id
-          setActiveAccountId(activeId)
-        }
+
+      // isActive: true is for the currently logged in/active account
+      const activeAcctArray = accounts.find(acct => acct.isActive)
+      if (activeAcctArray) {
+        const activeId = activeAcctArray[0].id
+        setActiveAccountId(activeId)
       }
     } catch (error) {
       event('multiAccount.retrieveAccounts.error', {error})
```

Comment:
I don't think hold up is necessary here, but do we want this to go to honeybadger where our other error reporting goes?  

Eventing will just go to snowflake, and I don't think the analytics folks are going to do anything with that.

Reply:
i will remove it in the new PR: https://github.com/influxdata/ui/pull/5810

Reply:
actually, where else should we report? i think we should report it to somewhere.

Reply:
Let me use this: `reportErrorThroughHoneyBadger`